src/fifelse.c

```diff
@@ -343,3 +343,194 @@ SEXP fcaseR(SEXP na, SEXP rho, SEXP args) {
   UNPROTECT(nprotect);
   return ans;
 }
+
+SEXP topnR(SEXP vec, SEXP n, SEXP dec) {
```

Comment:
why is it in `fifelse.c`? Seems `forder.c` would be a more natural place, otherwise its own C file should be fine

Reply:
Feel free to add it wherever you which. I just did not want to create too many files. There are already enough :-)